File: mybatis-plus-generator/src/test/java/com/baomidou/demo/enjoy/service/impl/SimpleServiceImpl.java
package com.baomidou.demo.enjoy.service.impl;

import com.baomidou.demo.enjoy.entity.Simple;
import com.baomidou.demo.enjoy.mapper.SimpleMapper;
import com.baomidou.demo.enjoy.service.ISimpleService;
import com.baomidou.mybatisplus.spring.service.impl.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 * 测试表 服务实现类
 * </p>
 *
 * @author baomidou
 * @since 2025-03-27
 */
@Service
public class SimpleServiceImpl extends ServiceImpl<SimpleMapper, Simple> implements ISimpleService {

}

File: mybatis-plus-generator/src/test/kotlin/com/baomidou/enjoy/service/impl/SimpleServiceImpl.kt
package com.baomidou.enjoy.service.impl;

import com.baomidou.enjoy.entity.Simple;
import com.baomidou.enjoy.mapper.SimpleMapper;
import com.baomidou.enjoy.service.ISimpleService;
import com.baomidou.mybatisplus.spring.service.impl.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 * 测试表 服务实现类
 * </p>
 *
 * @author baomidou
 * @since 2025-03-27
 */
@Service
open class SimpleServiceImpl : ServiceImpl<SimpleMapper, Simple>(), ISimpleService {

}
